### PR TITLE
the really big sprite food&drink sprite update thing

### PR DESCRIPTION
**Pull request by alsoandanswer**
*Changes sprites for a lot of food and drinks to match one consistent artstyle.

*also i tried removing the centre_of_mass thing on some sprites i hope i don't break anything
it compiled without errors so i got that going for me

*made some lame jokes in descriptions

forum thread : https://forums.aurorastation.org/topic/11553-big-ass-food-and-drink-resprite-project/

### wezzy changelog
i spent way too much time on this

please comment ty https://forums.aurorastation.org/topic/11180-wowzewows-spriter-application/

### CODE REVIEW

**Comment by avabee** [comment]
Check jdonut1 in food.dmi.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -685,8 +673,7 @@
 	name = "jelly donut"
 	desc = "You jelly?"
 	icon_state = "jdonut1"
-	filling_color = "#ED1169"
-	center_of_mass = list("x"=16, "y"=11)
+	filling_color = "#ED1169")
```
Comment (avabee) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: No parenthesis here.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -705,8 +692,7 @@
...
```
Comment (avabee) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Remove the parenthesis, please.

Comment (alsoandanswer) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: no problem

**Comment by alsoandanswer** [comment]
oh god oh fuck

**Comment by alsoandanswer** [comment]
🦀 CONFLICT IS GONE 🦀 
https://www.youtube.com/watch?v=Y5tjtUFL0j4&t=45

File: code/game/objects/items/trash.dm [hunk 3]
```diff
@@ -5,7 +5,7 @@
 /obj/item/trash
 	icon = 'icons/obj/trash.dmi'
 	w_class = 1.0
-	desc = "This is rubbish."
+	desc = "General waste material, refuse or litter. Dispose responsibly."
```
[the remark this answers is not in the document]
Comment (MarinaGryphon) [reply code/game/objects/items/trash.dm]: Yes. "Refuse" is another term for garbage, litter, waste material, trash, etc.

[the remark this answers is not in the document]
Comment (alsoandanswer) [reply code/game/objects/items/trash.dm]: yeah, it's refuse.

**Comment by Arrow768** [comment]
Reviews re-requested due to non transparent changes after awaiting merge

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**